Framework/Algorithms/src/CorrectKiKf.cpp

```diff
@@ -324,5 +290,19 @@ void CorrectKiKf::correctKiKfEventHelper(std::vector<T> &wevector,
   }
 }
 
+void CorrectKiKf::getEfixedFromParameterMap(double &Efi, int64_t i,
+                                            const SpectrumInfo &spectrumInfo,
+                                            const ParameterMap &pmap) {
+  if (spectrumInfo.isMonitor(i))
+    return;
```

Comment:
Probably not actually. This is not very nice in the way it works, but hopefully we will eventually have a better solution for the `EFixed` stuff that tidies this up.